### PR TITLE
What a year it's been

### PR DESCRIPTION
**Pull request by trobrock**

### CODE REVIEW

File: _posts/2020-07-11-what-a-year.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+That's a lot of really bad stuff. It'd be really easy to look at all those things
+and just want to throw in the towel on 2020. Just give up... Give up, what is that
+really? If my four-year-old was to ask me what "giving up" meant, how would I describe it?
+
+I think I'd describe it as admitting that it's too hard to continuing down your path.
+I'd describe it as changing direction away from something you really want. I would
+describe it as failing.
+
+I don't like failing, I like learning, but giving up is not learning. It's throwing
```
Comment (andrewkatz) [inline _posts/2020-07-11-what-a-year.md:19]: `I don't like failing. I like learning,`

File: _posts/2020-07-11-what-a-year.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+your hands up and walking away. How can I learn and move forward from what this year
+has dealt out so far? Let me look at what's gone well so far, and maybe there is some
+insight in there.
+
+First, I've bought a new house, that's always a good thing right? I've incorporated
```
Comment (andrewkatz) [inline _posts/2020-07-11-what-a-year.md:24]: `First, I bought a new house. That's always a good thing, right? I incorporated`

File: _posts/2020-07-11-what-a-year.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+my business with my business partner to set the stage for real estate investing
+(I've even made a few offers). I bought an RV, which is not something I would've
```
Comment (andrewkatz) [inline _posts/2020-07-11-what-a-year.md:26]: `I even made a few offers`

File: _posts/2020-07-11-what-a-year.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ever bought, or even thought I wanted, but it has been the experience of a lifetime.
+
+It's hard for me, especially in the moment, to look at the good in the bad, but when
```
Comment (andrewkatz) [inline _posts/2020-07-11-what-a-year.md:29]: `good in the bad. But when I do, I find`

File: _posts/2020-07-11-what-a-year.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+I do I find that there is generally something there. Even as I write this post I'm
+stewing through my head an opportunity to be a partner in a business that has gone
```
Comment (andrewkatz) [inline _posts/2020-07-11-what-a-year.md:31]: `stewing in my head about an opportunity`

File: _posts/2020-07-11-what-a-year.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+completely sideways of how I thought it'd go. When I look at the surface, all my work
+the last year has been wasted, I'm continually unable to get the position in a business
+I'm looking for, I could go on... But, when I look deeper I have to see the good,
+albeit as I type this I fail to see how it'll change things. Here it goes anyways.
+
+First, the opportunity to work on this project still exists, I've been working on
```
Comment (andrewkatz) [inline _posts/2020-07-11-what-a-year.md:37]: `I've been putting in so much effort to get the opportunity to actually *work* on this project for years now.`

File: _posts/2020-07-11-what-a-year.md (hunk 3 above)
Comment (trobrock) [reply _posts/2020-07-11-what-a-year.md:26]: How do I know when I should use I vs I've?

**Pull request merged**